prac_09/sort_files_v2.py

```diff
@@ -0,0 +1,32 @@
+"""
+CP1404/CP5632 Practical
+Program to sort files into directories based user input
+By Christopher Caferra
+"""
+
+import  os
+import shutil
+
+
+def main():
+    """Program to sort file extensions based on categories chosen by the user"""
+    extensions = {}
+    os.chdir('FilesToSort')
+    for filename in os.listdir('.'):        # save file names to list and extensions to dictionary
+        file_type = filename[filename.find('.') + 1:]
+        if filename.find(file_type):
```

Comment:
This is guaranteed to work unless it is a directory. Therefore `if not os.path.isdir(filename)` may be better to show you are checking for files not folders (this could be at the start as well which would prevent the need for any checker)